media-tv/kodi/files/kodi-17.3-unrar-vulnerability.patch

```diff
@@ -0,0 +1,45 @@
+See https://trac.kodi.tv/ticket/17510
```

Comment:
This summary line is too long which is cleanly visible as horrible wrapping. Please keep it down to 60-70 chars. Preferably short enough to include bugno.